mmv1/third_party/terraform/services/resourcemanager/fw_data_source_google_project.go

```diff
@@ -0,0 +1,158 @@
+package resourcemanager
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+
+	"github.com/hashicorp/terraform-provider-google/google/fwmodels"
+	"github.com/hashicorp/terraform-provider-google/google/fwtransport"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ datasource.DataSource              = &GoogleProjectFWDataSource{}
```

Comment:
Missed this on review of the previous framework datasources - is this actually necessary? My initial thought would be this actually doesn't do anything, and to satisfy interfaces we need to actually name a variable... Have you seen this pattern before?

Very quickly removing this from `data_source_provider_config_plugin_framework.go` and running `make build` doesn't throw a build error but I wonder if it would fail in runtime or something

Reply:
I had copied a lot of the base file from the Firebase data sources -- this could've been before the refactor if that removed it?